modules/cart/src/EventSubscriber/CartLanguage.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Drupal\commerce_cart\EventSubscriber;
+
+use Drupal\Core\Language\LanguageManager;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+/**
+ * Class OrderSubscriber.
+ */
+class CartLanguage implements EventSubscriberInterface {
+
+  /**
+   * The language manager.
+   *
+   * @var Drupal\Core\Language\LanguageManager
+   */
+  protected $languageManager;
+
+  /**
+   * CartLanguage constructor.
+   *
+   * @param Drupal\Core\Language\LanguageManager $language_manager
+   */
+  public function __construct(LanguageManager $language_manager) {
```

Comment:
Can we typehint on the interface here instead of the concrete class? ``Drupal\Core\Language\LanguageManagerInterface``